server-spi/src/main/java/org/keycloak/models/KeycloakSession.java

```diff
@@ -178,4 +178,11 @@
      */
     ThemeManager theme();
 
+    /**
+     * JWT manager
+     *
+     * @return
+     */
+    JWTManager jwt();
```

Comment:
Not a blocker, but as we discussed, a `token()` method and `TokenManager` (or similar) makes more sense. Or even just a `Token` or `KeycloakToken` interface which acts as a facade for any operation on keycloak tokens.